src/Billing/X125010837P.php

```diff
@@ -699,7 +699,7 @@ public static function genX12837P($pid, $encounter, &$log, $encounter_claim = fa
         }
 
         // Segment REF*F8 Payer Claim Control Number for claim re-submission.icn_resubmission_number
-        if (trim($claim->billing_options['icn_resubmission_number'] ?? null) > 3) {
+        if (strlen(trim($claim->billing_options['icn_resubmission_number'] ?? null)) > 3) {
```

Comment:
maybe the `?? null` isn't needed

Reply:
For php8, that deals with issue if $claim->billing_options['icn_resubmission_number'] is not a number (it will then set it to null). Otherwise will get a php typing error (trying to compare a non-number with a number). null can't break typing, so ok to compare a null with any type.
Is that null thing there breaking anything?

Reply:
Just realized you added the strlen function around this. In this case, should be able remove the `?? null` since strlen will return a 0 if it is empty.
(we should test it in php8 at some point in future to make sure)